ui/app/pages/swaps/fee-card/fee-card.js

```diff
@@ -26,6 +28,12 @@ export default function FeeCard({
     bestQuoteText = t('swapBetterQuoteAvailable');
   }
 
+  const getTranslatedNetworkName = () => {
+    const networkNameKey =
+      SWAPS_CHAIN_ID_TO_NETWORK_NAME_KEY_MAP[chainId] || 'networkNameEthereum';
+    return t(networkNameKey) || 'Ethereum'; // If there are no translated network names, it defaults to 'Ethereum'
```

Comment:
Nit: In general we should avoid passing variables to `t`, because our localized string linting script then has a more difficult time discovering whether the string is used or not.  There is a 'strict mode' that we've been slowly introducing one file at a time that assumes `t` is passed a string literal, so any use of a variable will make that task more difficult.

The alternative I'd suggest is deleting `SWAPS_CHAIN_ID_TO_NETWORK_NAME_KEY_MAP`, and replacing it with a function that accepts `t` and the chain ID, and returns the correct network name.

Reply:
> In general we should avoid passing variables to t, because our localized string linting script then has a more difficult time discovering whether the string is used or not
This is a good point, I always forget this.

> The alternative I'd suggest is deleting SWAPS_CHAIN_ID_TO_NETWORK_NAME_KEY_MAP, and replacing it with a function that accepts t and the chain ID, and returns the correct network name.
Wouldn't that still result in a variable passed to `t`?

Reply:
> Wouldn't that still result in a variable passed to t?

Ah, no, I see. The function would map chainId to calls to `t` using string literals

Reply:
So, there is https://github.com/i18next/react-i18next , which is one of the best internationalization frameworks in the world for React / React Native. Interpolation (passing dynamic values into translations) is one of the most used functionalities there: https://www.i18next.com/translation-function/interpolation

I believe we should support passing dynamic values into translations, since it's a very common thing to do. With i18next it's done like this:
```
{
    "key": "{{what}} is {{how}}"
}
i18next.t('key', { what: 'i18next', how: 'great' });
```

Reply:
We're not talking about interpolation here. Our translation helper does support interpolation.

Reply:
We've just clarified it over Slack - the issue is not about passing variables in the second param to `t`, but in the first one. Will create a function as suggested.